agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -948,3 +948,69 @@ func testV3TaskEndpointTags(t *testing.T, taskName, containerName, networkMode s
 	exitCode, _ := task.ContainerExitcode(containerName)
 	assert.Equal(t, 42, exitCode, fmt.Sprintf("Expected exit code of 42; got %d", exitCode))
 }
+
+func TestContainerOrdering(t *testing.T) {
```

Comment:
This should be included in the simple tests  (generated) since it isn't doing any complex validations here.